src/installer/corehost/cli/bundle/header.h

```diff
@@ -41,7 +41,8 @@ namespace bundle
     enum header_flags_t : uint64_t
     {
         none = 0,
-        netcoreapp3_compat_mode = 1
+        netcoreapp3_compat_mode = 1,
+        needs_native_libraries_extraction = 2,
```

Comment:
Do we really need a separate flag in the manifest for this? It seems like we'd be stuck with this oddly tailored flag (unless we rev the bundle version).

Could we just add the extraction path as long as it exists (we extract something)? I know we'd end up adding it unnecessarily in some cases (extracting symbols but not native libraries for example), but I don't think that is particularly risky.